qupath-core-processing/src/main/java/qupath/opencv/ml/PatchClassifierParams.java

```diff
@@ -267,36 +266,36 @@ private Builder(PatchClassifierParams params) {
 		
 		/**
 		 * Define the input channels using channel names.
-		 * @param channels
-		 * @return
+		 * @param channels The input channel names.
+		 * @return A modified builder.
```

Comment:
I find this misleading - should clarify it returns *this* builder.

A caller who has ever encountered a builder would expect it to be modified if we know that `this` is returned, but from the javadoc here it's not clear to me if builders are mutable or immutable. I wouldn't know if I *must* work with the returned builder (because it's different), rather than that I *can* use the returned builder (because it's the same object... just updated).